src/content/en/tools/service-worker-libraries/index.markdown

```diff
@@ -0,0 +1,138 @@
+---
+layout: shared/wide
+description: "Service Worker Libraries."
+title: "Service Worker Libraries"
+order: 6
+feedName: sw-toolbox Releases
+feedURL: https://github.com/googlechrome/sw-toolbox/releases.atom
```

Comment:
I think this is fine, we've got it for Web Starter Kit and the Polymer Starter Kit, so I'm ok with it.